node/bad_peer_test.go

```diff
@@ -127,7 +127,8 @@ func NewApp(conf *config.Config) (*App, error) {
 		WithConfig(conf),
 		WithLog(log.RegisterHooks(
 			log.NewWithLevel("", zap.NewAtomicLevelAt(zapcore.DebugLevel)),
-			events.EventHook())),
+			events.EventHook()),
+		),
```

Comment:
does your IDE do this, or you do this manually?
changes like this isn't very fruitful because the rest of the team dont really follow any guideline and they can be just as easily changed back in another PR.

Reply:
in this case it was my IDE. I'll see how to disable it

Reply:
We could think of using the same formatting preset backed up by some formatting tool by the whole team. IMHO, the codebase is a bit messy because there are no formatting rules. I'm OK with **anything** as long as it is consistent and allows autoformatting in IDE

Reply:
agreed. i don't care either as long as it's consistent.